src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -336,6 +321,57 @@ int64_t ClusterResourceScheduler::GetBestSchedulableNode(const TaskRequest &task
   return best_node;
 }
 
+int64_t ClusterResourceScheduler::GetBestSchedulableNode(const TaskRequest &task_req,
+                                                         bool actor_creation,
+                                                         bool force_spillback,
+                                                         int64_t *total_violations,
+                                                         bool *is_infeasible) {
+  // The zero cpu actor is a special case that must be handled the same way by all
+  // scheduling policies.
+  if (actor_creation && task_req.IsEmpty()) {
+    int64_t best_node = -1;
+    // This an actor which requires no resources.
+    // Pick a random node to to avoid all scheduling all actors on the local node.
```

Comment:
Supernit:
```suggestion
    // This is an actor which requires no resources.
    // Pick a random node to to avoid scheduling all actors on the local node.
```